app/src/main/java/org/coepi/android/ui/symptoms/cough/CoughDurationFragment.kt

```diff
@@ -29,4 +28,9 @@ class CoughDurationFragment : Fragment() {
         onBack { viewModel.onBack() }
         toolbar.setNavigationOnClickListener { viewModel.onBackPressed() }
     }.root
+
+    override fun onStop() {
+        KeyboardHider().hideKeyboard(this.requireContext(), this.requireView())
```

Comment:
No unnecessary (not required by the compiler) `this` please 🙂

Reply:
Ah, completely forgot about that with Kotlin!